.gitignore

```diff
@@ -0,0 +1,28 @@
+# Игнорировать временные файлы и директории
+node_modules/
+build/
+.DS_Store
+
+# Игнорировать файлы окружения (если используете .env)
+.env
+.env.local
+.env.development
+.env.test
+.env.production
```

Comment:
should be added to git repo, except .local (probably will be ignored by default)